### ISSUE TITLE
Requirements: psycopg2-binary to psycopg-binary

### ISSUE THREAD

**Issue by catbyte-io** [adilmohak/django-lms#49]
Hello. As Psycopg2 will soon be deprecated, should we switch to using Psycopg3 (AKA Psycopg)?

https://pypi.org/project/psycopg-binary/

**Comment by adilmohak** [comment]
Hi, we're now using psycopg2 instead of psycopg-binary on both development and production environment

**Comment by adilmohak** [comment]
Can you provide a link related to the deprecation of psycopg2

**Comment by catbyte-io** [comment]
Yes, the mention of psycopg2 to be most likely deprecated soon is mentioned on this page of Django release notes.
https://django.readthedocs.io/en/stable/releases/4.2.html

**Comment by catbyte-io** [comment]
I would be happy to work on changing this if this is something you would like to do. I am new to contributing, so I am trying to learn with projects like Django that I am familiar with. Thank you for considering!

**Comment by adilmohak** [comment]
For the project to run at first glance we don't need psycopg2 anymore
